docs/app/docs/guides/pinning_packages.md

```diff
@@ -17,21 +17,21 @@ Because the repository changes frequently, and new releases of Nixpkgs infrequen
 You can look up the available versions of a package by running `devbox search <package_name>`. For example, to see the available versions of `python`, you can run `devbox search python`:
 
 ```bash
-$ devbox search nodejs
-
-Found 168+ results for "nodejs":
-
-* nodejs (19.8.1, 19.7.0, 19.5.0, 19.2.0, 18.16.0, 18.15.0, 18.14.2, 18.13.0, 18.12.1, 18.10.0,
-18.8.0, 18.4.0, 18.0.0, 17.9.0, 17.5.0, 17.3.0, 17.0.1, 16.19.1, 16.19.0, 16.18.1, 16.17.1, 16.17.0,
-16.15.0, 16.14.0, 16.13.1, 16.13.0, 16.8.0, 16.4.0, 16.0.0, 15.14.0, 15.10.0, 15.5.0, 15.0.1,
-14.18.1, 14.18.0, 14.17.5, 14.17.1, 14.16.1, 14.16.0, 14.15.3, 14.15.0, 14.9.0, 14.4.0, 13.14.0,
-12.22.12, 12.22.10, 12.22.8, 12.22.7, 12.22.5, 12.22.1, 12.21.0, 12.20.0, 12.19.0, 12.18.3,
-12.18.1, 10.24.1, 10.24.0, 10.23.0, 10.22.0, 10.21.0)
-* nodejs_16 (16.20.0)
-* nodejs_18 (18.16.0)
-* nodejs_19 (19.9.0)
-* nodejs_20 (20.0.0)
-...
+$ devbox search python
```

Comment:
@Lagoja did you intend to include the nodejs results before, or was it supposed to be python?

Reply:
I think it should be nodejs for this example, since the results look cleaner